core/includes/date.inc

```diff
@@ -1611,17 +1628,17 @@ function date_increment_round(&$date, $increment) {
     $second = intval(round(intval(date_format($date, 's')) / $increment) * $increment);
     $minute = intval(date_format($date, 'i'));
     if ($second == 60) {
-      $minute += 1;
+      ++$minute;
```

Comment:
I'm curious: what's the benefit of this notation change? (Also below)

Reply:
You're right! That's Squiz.Operators.IncrementDecrementUsage. Sorry for the noise, then. :+1: